transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/PatientsSetSpec.groovy

```diff
@@ -169,4 +169,25 @@ class PatientsSetSpec extends RESTSpec {
 //        assert responseData.type == 'AccessDeniedException'
 //        assert responseData.message == "Access denied to patient set or patient set does not exist: ${setID.id}"
     }
+
+    /**
+    *  when: "I try to fetch all patientSets"
+    *  then: "the list of all patientSets is returned"
+    */
+    def "get list of patientSets"() {
+        def request = [
+                path      : PATH_PATIENT_SET,
+                acceptType: contentTypeForJSON,
+                statusCode: 400
+        ]
+
+        when: "I try to fetch all patientSets"
+        def responseData = get(request)
+
+        then: "the list of all patientSets is returned"
+        assert responseData.patientSets != null
+        responseData.patientSets.each {
```

Comment:
This should also check that the list of patient sets is not empty. But to guarantee that the test also needs to create a patient set